app/views/devise/registrations/new.html.erb

```diff
@@ -7,20 +7,22 @@
 
         <h2>Sign up</h2>
 
-        <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'form-horizontal' }) do |f| %>
-          <%= devise_error_messages! %>
-          <% if ENV['ON_HEROKU'] && User.count.zero? %>
-          <div class="heroku-instructions">
-            <% app_name = request.host[/\A[^.]+/] %>
-            <p>If you are the owner of this application, take the following steps to complete the setup:</p>
+        <% if User.allow_signup? %>
```

Comment:
Looks ugly in github, but this is really just an extra indent for everything on the page in the signup `<div>`.